src/components/RecipeCard.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { TimeIcon } from '@chakra-ui/icons';
+import Img from 'gatsby-image';
+import { graphql } from 'gatsby';
+
+interface Props {
+  data: GatsbyTypes.RecipeCardFragment;
+}
+
+function RecipeCard({ data }: Props): JSX.Element {
+  return (
+    <div style={{padding: '50px 10px 50px 10px', marginRight: '25px'}}>
+      <div style={{ marginBottom: '5%'}}>
```

Comment:
So for this top bar div, I would recommend using Chakra UI's `Flex` component so we don't have to worry about setting `position` on the children

See the bottom of https://chakra-ui.com/docs/layout/flex for an example of this